packages/bpk-component-popover/src/BpkPopoverPortal-test.js

```diff
@@ -71,7 +73,8 @@ describe('BpkPopoverPortal', () => {
       .create(
         <BpkPopoverPortal
           id="my-popover"
-          target={() => document.createElement('button')}
+          // $FlowFixMe
+          target={() => document.createElement('button', { type: 'button' })}
```

Comment:
I think there should be a quick note on this comment to explain why we're opting out of Flow here? Same as the convention we have to justify eslint opt outs.

Reply:
Agreed - ill add it here and in the code for brevity:

"Flow doesn't yet support a secondary argument for document.createElement".

> Same as the convention we have to justify eslint opt outs.

Where is this convention? :P